.expeditor/config.yml

```diff
@@ -95,6 +72,29 @@ subscriptions:
       - built_in:publish_rubygems
       - built_in:promote_habitat_packages
       - built_in:notify_chefio_slack_channels
+  # These actions are taken, in order they are specified, anytime a Pull Request is merged.
```

Comment:
It appears this project wants an empty space between subscriptions. Please add an empty line above the comment.